code/modules/blob/blob.dm

```diff
@@ -139,10 +53,17 @@
 	else
 		icon_state = "blob_damaged"
 
+/obj/effect/blob/process()
+	regen()
+	if(world.time < next_attack + attack_time)
```

Comment:
Next attack is a shit name for this var. If you want to use that name, make it so that `next_attack = world.time + attack_time` and check `if (world.time < next_attack)`.